Sources/CorePayments/AnalyticsEventData.swift

```diff
@@ -18,6 +18,7 @@ struct AnalyticsEventData: Encodable {
         case clientOS = "client_os"
         case component = "comp"
         case deviceManufacturer = "device_manufacturer"
+        case environment = "merchant_app_environment"
```

Comment:
So, we can't overwrite the `env` param since it gets set by the FPTI service based on the host URL.

I opted to make the name of this really explicit to `merchant_app_environment` so we can distinguish it from the FPTI environment, since we are using FPTI live for both live/sand and prod events.

Would love folks feedback on the best name for this tag to make it extra clear. Either way, this is a new tag we will need to submit for tag approval so the world is our oyster for names.

Reply:
I wonder if "merchant_configuration_environment" would be more clear since it's passed in the `CoreConfig` object?

Reply:
Yeh, that's a good call. 

Also worth noting - this value will also be used for differentiating BT SDK coming from Arachne.

So it'll go:
* Events from https://origin-analytics-prod.sandbox.braintree-api.com --> https://api.paypal.com/v1/tracking + `new_tag_name = sandbox`
* Events from https://origin-analytics-sand.sandbox.braintree-api.com --> + `new_tag_name = production`

Reply:
We _technically_ can sent `environment = whatever`, but I figured doing that would be confusing since a pre-populated `env` will always be set to `live`.

Reply:
Yeah I think that makes sense. Are you saying that this event is how we will tell BT Arachne vs BT FPTI events eventually (when we port Arachne to FPTI automatically)? I may have misunderstood that portion of your comment so want to make sure I followed what you were saying.

Reply:
cc: @DPoplin @debrado